### PR TITLE
Update IDL files to integrate change for logging disabling. 

### PR DESCRIPTION
**Pull request by phaazon**
The version number will be bumped in a specific commit according to a discussion with @KhalilBellakrid.

### CODE REVIEW

File: core/src/wallet/pool/api/WalletPoolApi.cpp [hunk 1]
```diff
@@ -51,8 +51,10 @@ namespace ledger {
                                      const std::shared_ptr<api::ThreadDispatcher> &dispatcher,
                                      const std::shared_ptr<api::RandomNumberGenerator> &rng,
                                      const std::shared_ptr<api::DatabaseBackend> &backend,
-                                     const std::shared_ptr<api::DynamicObject> &configuration) {
-            auto pool = ledger::core::WalletPool::newInstance(name, Option<std::string>(password), httpClient, webSocketClient, pathResolver, logPrinter, dispatcher, rng, backend, configuration);
+                                     const std::shared_ptr<api::DynamicObject> &configuration,
+                                     bool disableLogging) {
+            std::cout << "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\nfrom static newInstance " << disableLogging << std::endl;
```
Comment (KhalilBellakrid) [inline core/src/wallet/pool/api/WalletPoolApi.cpp]: Nice logs BTW :D !

**Comment by phaazon** [comment]
According to a live discussion, I’m going to change a little bit the way this is done so that we the public interface doesn’t break.

File: core/src/wallet/pool/api/WalletPoolApi.cpp (hunk 1 above)
Comment (phaazon) [reply core/src/wallet/pool/api/WalletPoolApi.cpp]: It’s on purpose! :trollface:

**Comment by phaazon** [comment]
PTAL :)

File: core/src/wallet/pool/WalletPool.cpp [hunk 2]
```diff
@@ -79,13 +79,15 @@ namespace ledger {
...
             // Logger management
+            auto enableLogger = _configuration->getBoolean(api::PoolConfiguration::ENABLE_INTERNAL_LOGGING).value_or(true);
             _logger = logger::create(
                     name + "-l",
                     password.toOptional(),
                     dispatcher->getSerialExecutionContext(fmt::format("logger_queue_{}", name)),
                     pathResolver,
                     logPrinter,
-                    disableLogging
+                    logger::DEFAULT_MAX_SIZE,
```
Comment (KhalilBellakrid) [inline core/src/wallet/pool/WalletPool.cpp]: Good old memories ! :trollface:

Comment (phaazon) [reply core/src/wallet/pool/WalletPool.cpp]: 😆

**Comment by phaazon** [comment]
This whole PR is completely ups and downs. I’m going surgery and will re-create a PR (the name of the PR doesn’t make any sense anyway.).

**Pull request closed without merging**